lang/src/accounts/signer.rs

```diff
@@ -47,7 +46,7 @@ impl<'info> Signer<'info> {
 
     /// Deserializes the given `info` into a `Signer`.
     #[inline(never)]
-    pub fn try_from(info: &AccountInfo<'info>) -> Result<Signer<'info>, ProgramError> {
+    pub fn try_from(info: &AccountInfo<'info>) -> anchor_lang::Result<Signer<'info>> {
```

Comment:
Note that you can just use `Result<T>`everywhere instead of `anchor_lang::Result<T>`.